### PR TITLE
PAYARA-2232 Added link to application.wadl

### PR DESCRIPTION
**Pull request by michaelranaldo**
Diff is a little weird due to added loop. application.wadl is [autogenerated by jersey](https://jersey.github.io/documentation/latest/wadl.html#d0e13102) so should always be located here.

### CODE REVIEW

**Comment by michaelranaldo** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
jenkins test please

File: appserver/admingui/common/src/main/resources/logViewer/logViewer.jsf [hunk 1]
```diff
@@ -48,10 +48,9 @@
     setResourceBundle(key="i18n" bundle="org.glassfish.admingui.core.Strings");
     setAttribute(key="logentries" value=[]);
 </ui:event>
-
 <sun:page id="page1">
 <ui:event type="beforeCreate">
-    <!-- setPageSessionAttribute(key="instanceName" value="server"); -->
+    <!-- setPageSessionAttribute(key="instanceName" netbeans changeavlue="server"); -->
```
Comment (Cousjava) [inline appserver/admingui/common/src/main/resources/logViewer/logViewer.jsf]: What is the point of this change?

File: nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/LogFormatHelper.java [hunk 2]
```diff
@@ -38,42 +38,44 @@
  * holder.
  */
 // Portions Copyright [2016] [Payara Foundation and/or its affiliates]
```
Comment (Cousjava) [inline nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/LogFormatHelper.java]: Year

**Comment by michaelranaldo** [comment]
I've pushed my stashes to save from PC formatting, this will be reverted

**Comment by michaelranaldo** [comment]
Fixed commits

**Comment by smillidge** [comment]
jenkins test please

**Pull request merged**